src/util/include/util/SaveNameUtils.h

```diff
@@ -9,9 +9,6 @@
  * @license GNU GPLv2 or later
  */
 
-#pragma once
```

Comment:
`#pragma once` should still be in the header file.

Reply:
Yes, was already confused about that one, sorry